packages/block/src/from-rpc.ts

```diff
@@ -49,11 +50,13 @@ export function blockFromRpc(
   const header = blockHeaderFromRpc(blockParams, options)
 
   const transactions: TypedTransaction[] = []
-  const opts = { common: header._common }
-  for (const _txParams of blockParams.transactions) {
-    const txParams = normalizeTxParams(_txParams)
-    const tx = TransactionFactory.fromTxData(txParams as TxData, opts)
-    transactions.push(tx)
+  if (blockParams.transactions !== undefined) {
+    const opts = { common: header._common }
+    for (const _txParams of blockParams.transactions) {
```

Comment:
```suggestion
    const opts = { common: header._common }
    for (const _txParams of blockParams.transactions ?? []) {
```
not a big deal, but one less line of code